src/main/java/com/yandex/money/api/model/showcase/components/Component.java

```diff
@@ -8,6 +10,13 @@
  */
 public abstract class Component {
 
+    @Override
+    public String toString() {
+        return getToStringBuilder().toString();
+    }
+
+    protected abstract ToStringBuilder getToStringBuilder();
```

Comment:
I think it should be extracted to an interface. Therefore we could use it in other classes.